lib/layout.js

```diff
@@ -113,6 +114,27 @@ const PodiumLayout = class PodiumLayout {
             value: new Metrics(),
         });
 
+        // Skip a tick to ensure the metric stream has been consumed
+        setImmediate(() => {
```

Comment:
what happens when you don't do this?

Reply:
The metric is lost we are inside the constructor